src/index.html

```diff
@@ -1,57 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Code editor created by the Raspberry Pi Foundation"
-    />
+    <meta name="description" content="Code editor created by the Raspberry Pi Foundation" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
+        manifest.json provides metadata used when your web app is installed on a
+        user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+      -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+        Notice the use of %PUBLIC_URL% in the tags above.
+        It will be replaced with the URL of the `public` folder during the build.
+        Only files inside the `public` folder can be referenced from the HTML.
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+        Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+        work correctly both with client-side routing and a non-root public URL.
+        Learn how to configure a non-root public URL by running `npm run build`.
+      -->
     <!-- Google Tag Manager -->
-    <script data-cookieconsent='statistics' type='text/plain'>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Just wondering why we need to remove this?

Reply:
Ohhh I see... Will that affect the consent for other things in GTM in any way, or will the CookieBot/GTM integration handle that itself?

Reply:
So most of the tags are GA-4 which just works and needs me to click the defaults are good. We'll need to think about Hotjar before enabling 😄 
![image](https://user-images.githubusercontent.com/74183390/232527564-3862e344-26e0-4591-9a5e-ef15f4018837.png)